plugins/dsrhub_grpc/dsrhub_grpc.go

```diff
@@ -28,6 +29,7 @@ type DsrHubGRPCConfig struct {
 	URL     string                      `json:"url"`
```

Comment:
nit: s/Dsr/DSR/g I think in go it's idiomatic to have capital letters for initials.